### ISSUE TITLE
Pandoc error during re-building of vignette outputs on Windows

### ISSUE THREAD

**Issue by sjmgarnier** [sjmgarnier/viridis#16]
Error message returned during package check is: 
```
* checking re-building of vignette outputs ... NOTE
Error in re-building vignettes:
  ...
Loading required package: raster
Loading required package: sp

Attaching package: 'raster'

The following object is masked from 'package:colorspace':

    RGB

Loading required package: lattice
Loading required package: latticeExtra
Loading required package: RColorBrewer

Attaching package: 'latticeExtra'

The following object is masked from 'package:ggplot2':

    layer

Warning: running command '"d:/compiler/pandoc/pandoc" +RTS -K512m -RTS intro-to-viridis.utf8.md --to html --from markdown+autolink_bare_uris+ascii_identifiers+tex_math_single_backslash-implicit_figures --output intro-to-viridis.html --smart --email-obfuscation none --self-contained --standalone --section-divs --table-of-contents --toc-depth 1 --template "d:\RCompile\CRANpkg\lib\3.3\rmarkdown\rmd\h\default.html" --css "d:\RCompile\CRANpkg\lib\3.3\rmarkdown\rmarkdown\templates\html_vignette\resources\vignette.css" --mathjax --variable "mathjax-url:https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" --highlight-style pygments' had status 6
Error: processing vignette 'intro-to-viridis.Rmd' failed with diagnostics:
pandoc document conversion failed with error 6
Execution halted
```

**Comment by sjmgarnier** [comment]
Note that I am not able to reproduce this error on Mac OS X 10.10.5 and Ubuntu 15.04 with latest versions of R, devtools, and pandoc for each platform respectively.

**Comment by sjmgarnier** [comment]
Note also that this was flagged by Uwe Ligges from CRAN when checking our package submission.

**Comment by hrbrmstr** [comment]
I can fire up a Windows 10 system tomorrow and attempt to see what's up. It _may_ just be that the vignette size is too large (that causes pandoc build errors).

If you have time, you can try to chop down the vignette and then use `devtools::build_win()` to submit it to winbuilder (and it'll return links to errors/etc).

**Comment by sjmgarnier** [comment]
@hrbrmstr I don't have time tonight, but I can try tomorrow. Thanks for the tip.

**Comment by noamross** [comment]
I just ran `build_win()` but since @sjmgarnier is the maintainer in the DESCRIPTION file, the report will be sent to his e-mail.

**Comment by sjmgarnier** [comment]
@noamross Ok. Thanks for this. I'll forward it to your gmail address (I imagine this is you: http://www.noamross.net/connect.html).

**Comment by sjmgarnier** [comment]
@noamross Actually, that was pretty fast. Here is the link to the report: http://win-builder.r-project.org/1Zr74HKNn31h/. It will expire in 3 days.

**Comment by sjmgarnier** [comment]
@noamross And here is a second report: http://win-builder.r-project.org/nL54BeidcjIo/. I guess you made changes between the first and second reports. Error seems to be always the same. 

Note that in a separate email exchange with Uwe Ligges, he told me to move ahead with resubmitting the package after making minor changes to the format of the URLs in the vignette (apparently they're not following CRAN standards), and that he'll try to debug this problem when he can find the time.

**Comment by hrbrmstr** [comment]
ah. if you referenced other pkgs with the direct URLs and not the `?` URLs they dislike that ;-)

**Comment by sjmgarnier** [comment]
@hrbrmstr Yes, they do indeed. But I doubt this is the cause of the problem here unfortunately. Anyway, it seems that Uwe Ligges will approve the package despite this once the URLs are corrected. I can't do it tonight, but I will do it first thing tomorrow.
